README.md

```diff
@@ -0,0 +1,82 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Budget_App
+
+> This project is created as part of the rails capstone at Microverse. This app is a  web application where a user manage its budget: have a list of transactions associated with a category, so that the user can see how much money it spent and on what.
+
+
+## Built With
+
+- Major: Ruby, HTML,CSS
+- Frameworks: `Ruby on Rails`, `Bootstrap5`
+- Libraries: `Cancancan`, `Devise`
+- Technologies: Git, Gitflow, Linters, Github, PostgreSql, Rspec, Capybara, Robocop
+
+## Live Demo
+
+[Live Demo Link](https://budget-controller1.herokuapp.com/) 
```

Comment:
- [ ] As the previous reviewer stated, kindly make sure your live demo work properly, I think this problem happened because you didn't run the migration in Heroku, you can watch [this video](https://www.youtube.com/watch?v=bQ2wXZf9irA) at `6:00` for the solution for this issue.